tests/test_literals.py

```diff
@@ -1,9 +1,10 @@
 """Test RDF literals."""
+# pylint: disable=invalid-name
 
 
 def test_string() -> None:
     """Test creating a string literal."""
-    from tripper.triplestore import Literal
+    from tripper import Literal
```

Comment:
```suggestion
    from tripper.literal import Literal
```
Why use a general import rather than the explicitly from the module we are testing?

This goes for all the changed imports here.

Reply:
Lets address this when we have agreed on the next question